app/Phragile/StatusByStatusFieldDispatcher.php

```diff
@@ -9,17 +9,18 @@ public function __construct($reviewTagPHID)
 		$this->reviewTagPHID = $reviewTagPHID;
 	}
 
-	private function isTaskInReview(array $task)
+	private function isTaskInReview(Task $task)
 	{
-		return !$this->isClosed($task) && in_array($this->reviewTagPHID, $task['attachments']['projects']['projectPHIDs']);
+		return !$this->isClosed($task) && in_array($this->reviewTagPHID, $task->getProjectPHIDs());
 	}
 
-	private function isTaskBeingDone(array $task)
+	// TODO: isDone($task)?
```

Comment:
I realize now how terrible the current name is. It actually mean that the task is in doing. So `isDone` is wrong but `isInDoing` might be better than the current version?